ambari-server/src/main/resources/stack-hooks/before-START/scripts/params.py

```diff
@@ -96,7 +96,7 @@
 java_exec = "{0}/bin/java".format(java_home) if java_home is not None else "/bin/java"
 
 #users and groups
-has_hadoop_env = 'hadoop-env' in module_configs
+has_hadoop_env = module_configs.get_all_properties(module_name, "hadoop-env") is not None
```

Comment:
get_all_properties() never returns None. It returns an empty map if the config-type does not exist in the command.json